core/src/main/scala/ai/h2o/sparkling/backend/H2OFrameRelation.scala

```diff
@@ -26,13 +26,16 @@ import org.apache.spark.sql.types.{Metadata, MetadataBuilder, StructField, Struc
 import org.apache.spark.sql.{Row, SQLContext}
 
 /**
- * H2O relation implementing column filter operation.
- */
+  * H2O relation implementing column filter operation.
+  */
 case class H2OFrameRelation(frame: H2OFrame, copyMetadata: Boolean)(@transient val sqlContext: SQLContext)
-  extends BaseRelation with TableScan with PrunedScan {
```

Comment:
If I'm the original indentation is used in Spark as well (two spaces)

Reply:
It I'm not wrong scalafmt has 2 spaces by default, do you prefer different value?

Reply:
Example: https://github.com/apache/spark/blob/8643e5d9c50294f59b01988d99d447a38776178e/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala#L46

Reply:
Ah, we have 4 spaces! Will fix

Reply:
Is it this rule? https://scalameta.org/scalafmt/docs/configuration.html#continuationindentdefnsite

Reply:
4 spaces are used only for class and method parameters.

Reply:
I'm lost a little bit. So what should I do? :D You prefer 2 spaces everywhere?

Reply:
Example:
https://github.com/apache/spark/blob/6704103499d2003b1879ff0b4b8e29141e401b9f/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala#L2271
but it's up to us. We can select the way which is more convenient for us.

Reply:
Yup, switched to 2, it makes it easier to read. Please have additional look :)